tests/utils/datatypes/datatype_tests.py

```diff
@@ -38,6 +38,30 @@
 # python manage.py test tests.utils.datatypes.datatype_tests --settings="tests.test_settings"
 
 
+class BaseDataTypeTests(ArchesTestCase):
+    def test_get_tile_data_only_none(self):
```

Comment:
Adding this test seems out of scope for this ticket?

Reply:
Yeah, sorry for the potentially confusing diff - I just moved that test up to the top because IMO it makes sense to have the tests for the Base class first, then datatype-specific tests after. Previously that test was under `BooleanDataTypeTests`.

Reply:
https://github.com/archesproject/arches/pull/11946/commits/e149e70ed812e7439fe4afc6d616116abb27de56